metaci/cumulusci/tasks.py

```diff
@@ -0,0 +1,19 @@
+from django import db
+from django_rq import job
+
+from metaci.cumulusci.models import ScratchOrgInstance
+
+
+@job('short')
+def prune_orgs():
+    db.connection.close()
+    pruned = []
+    for soi in ScratchOrgInstance.expired.all():
+        soi.delete()
```

Comment:
This actually deletes the row since the model is a standard Django model. Did you mean to use `soi.delete_org()` instead?

Reply:
whoops good catch yes, originally I had looked at switching SOI to a softdeletemodel but the mechanics of scratch orgs are a little weird, so it didn't make sense in the end.